addons/hw_drivers/tools/helpers.py

```diff
@@ -20,7 +19,7 @@
 import socket
 import subprocess
 from urllib.parse import parse_qs
-import urllib3
+import urllib3.util
```

Comment:
what is the point of this import ?

Reply:
we need it to use the method: `urllib3.util.parse_url()` on line 642.